go.mod

```diff
@@ -5,50 +5,47 @@ go 1.13
 require (
 	github.com/200sc/bebop v0.0.8
 	github.com/DataDog/zstd v1.4.5 // indirect
-	github.com/Hatch1fy/errors v0.1.0 // indirect
 	github.com/Sereal/Sereal v0.0.0-20190618215532-0b8ac451a863
 	github.com/alecthomas/binary v0.0.0-20171101084825-6e8df1b1fb9d
-	github.com/andyleap/gencode v0.0.0-20171124163308-e1423834d4b4 // indirect
-	github.com/andyleap/parser v0.0.0-20160126201130-db5a13a7cd46 // indirect
 	github.com/calmh/xdr v1.1.0
 	github.com/davecgh/go-xdr v0.0.0-20161123171359-e6a2ba005892
 	github.com/glycerine/go-capnproto v0.0.0-20190118050403-2d07de3aa7fc
+	github.com/glycerine/goconvey v0.0.0-20190410193231-58a59202ab31 // indirect
 	github.com/glycerine/rbtree v0.0.0-20190406191118-ceb71889d809 // indirect
 	github.com/gogo/protobuf v1.2.1
 	github.com/golang/protobuf v1.3.2
-	github.com/golang/snappy v0.0.1 // indirect
 	github.com/google/flatbuffers v1.11.0
-	github.com/gorilla/websocket v1.4.1 // indirect
+	github.com/gopherjs/gopherjs v0.0.0-20210603182125-eeedf4a0e899 // indirect
 	github.com/hprose/hprose-go v0.0.0-20161031134501-83de97da5004
 	github.com/hprose/hprose-golang v2.0.4+incompatible
 	github.com/ikkerens/ikeapack v1.5.1
 	github.com/itsmontoya/mum v0.3.2
 	github.com/json-iterator/go v1.1.7
-	github.com/kr/pretty v0.1.0 // indirect
 	github.com/linkedin/goavro v2.1.0+incompatible
 	github.com/mailru/easyjson v0.0.0-20190626092158-b2ccc519800e
 	github.com/minio/sha256-simd v0.1.0 // indirect
 	github.com/niubaoshu/gotiny v0.0.3
-	github.com/pascaldekloe/colfer v1.7.2 // indirect
-	github.com/pascaldekloe/name v1.0.0 // indirect
 	github.com/philhofer/fwd v1.0.0 // indirect
-	github.com/pkg/errors v0.8.1 // indirect
 	github.com/protolambda/zssz v0.1.1 // indirect
 	github.com/prysmaticlabs/go-bitfield v0.0.0-20190825002834-fb724e897364 // indirect
 	github.com/prysmaticlabs/go-ssz v0.0.0-20190827151743-72881c4223d8
-	github.com/shamaton/msgpack v1.1.1 // indirect
 	github.com/shamaton/msgpack/v2 v2.0.0
 	github.com/shamaton/msgpackgen v0.1.1
+	github.com/stretchrcom/testify v1.7.1 // indirect
 	github.com/tinylib/msgp v1.1.0
-	github.com/ttacon/chalk v0.0.0-20160626202418-22c06c80ed31 // indirect
 	github.com/ugorji/go/codec v1.1.7
 	github.com/valyala/fastjson v1.6.3
 	github.com/vmihailenco/msgpack/v4 v4.3.0
-	github.com/ymz-ncnk/musgo v0.1.6 // indirect
+	github.com/ymz-ncnk/musgo v0.1.6
 	go.dedis.ch/protobuf v1.0.6
+	go.mongodb.org/mongo-driver v1.5.1
 	golang.org/x/sys v0.0.0-20210616094352-59db8d763f22 // indirect
-	golang.org/x/tools v0.1.3 // indirect
 	gopkg.in/linkedin/goavro.v1 v1.0.5
 	gopkg.in/mgo.v2 v2.0.0-20190816093944-a6b53ec6cb22
 	zombiezen.com/go/capnproto2 v2.17.0+incompatible
 )
+
+replace (
+	github.com/itsmontoya/mum v0.5.6 => github.com/mojura/enkodo v0.5.6
```

Comment:
Mmm annoying :\

Reply:
Yeah ran into problems - these libs renamed themselves. Updating to the latest caused other issues.